.github/workflows/sentry_resque_test.yml

```diff
@@ -53,17 +58,10 @@ jobs:
       - name: Run specs
         env:
           RUBYOPT: ${{ matrix.options.rubyopt }}
-        run: |
-          bundle install --jobs 4 --retry 3
-          bundle exec rake
+        run: bundle exec rake
 
       - name: Run specs with Rails
-        env:
-          BUNDLE_GEMFILE: Gemfile_with_rails.rb
-          RUBYOPT: ${{ matrix.options.rubyopt }}
-        run: |
-          bundle install --jobs 4 --retry 3
-          bundle exec rake
+        run: bundle exec rake
```

Comment:
We can remove this step as the previous one would already run with Rails.

Reply:
@st0012 I made it so that it runs with and without Rails, as I reckon that was the original intention. WDYT?